lib/deploy/firestore/prepare.js

```diff
@@ -21,7 +21,11 @@ function _prepareRules(context, options) {
 }
 
 function _prepareIndexes(context, options) {
-  return context.firestoreIndexes ? firestoreIndexes.prepare(context, options) : RSVP.resolve();
+  if (!(context.firestoreIndexes && options.config.get('firestore.indexes'))) {
```

Comment:
This negation is making my eyes cross, please rewrite as:

    if (!context.firestoreIndexes || !options.config.get('firestore.indexes'))